torch/nn/quantized/modules/linear.py

```diff
@@ -229,16 +229,9 @@ def from_float(cls, mod):
         act_scale, act_zp = activation_observer.calculate_qparams()
         assert weight_observer.dtype == torch.qint8, 'Weight observer must have dtype torch.qint8'
         wt_scale, wt_zp = weight_observer.calculate_qparams()
-        # Scale bias to activation_scale/2^16, this quantizes bias
-        # to about 24 bits of precision
-        bias_scale = float(act_scale / (2**16))
```

Comment:
The bias handler logic in the new FBGEMM API is different from "activation_scale/2^16" here. Will this cause any numerical diffenrece?

Reply:
Just saw your comment that the bias is just not quantized, so I guess the accuracy should be improved with the new API.

Reply:
This was a temporary change, any numeric change will be below 1 LSB as bias was previously quantized to ~24 bit precision.